components/Alert/alert.css

```diff
@@ -21,21 +21,21 @@
 }
 .outline-primary {
   border: 1px solid var(--primary-color);
-  background-color: white;
+  background-color: var(--white-color);
   color: var(--primary-color);
 }
 .outline-success {
```

Comment:
For readability, it is a good practice to leave one blank line between two code-blocks